src/screens/fareResults.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import { ScreenNavigationProps } from '../routes';
+import { config } from '../config';
+import FlatListComponent from '../components/flatList';
+import { useEffect } from 'react';
+import { FareResponse, Journey } from '../model/fareResponseClass';
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#eee',
+  },
+  text: {
+    paddingBottom: 24,
+  },
+});
+
+type FareResultsProps = ScreenNavigationProps<'FareResults'>;
+
+const getTrainFares = async (
+  departStation: string | null,
+  arriveStation: string | null,
```

Comment:
Probably worth being extra explicit with these, e.g. `arrivalStationCrsCode` so that someone doesn't call this method with "York" as an argument and expect it to work